analyzers/tests/SonarAnalyzer.UnitTest/Helpers/SyntaxHelperTest.cs

```diff
@@ -98,19 +98,23 @@ public void NameIs_VB()
             toString.NameIs(null).Should().BeFalse();
         }
 
-        [TestMethod]
-        public void IsAssignmentToTuple_IsTrue()
+        [DataTestMethod]
+        [DataRow("($$a, b) = (42, 42);")]
+        [DataRow("(a, $$b) = (42, 42);")]
+        [DataRow("(a, (b, $$c)) = (42, (42, 42));")]
+        [DataRow("(a, (b, ($$c, d))) = (42, (42, (42, 42)));")]
+        public void IsAssignmentToTuple_IsTrue(string assignment)
```

Comment:
should the name of the test be `IsInTupleAssignmentTarget_IsTrue`?

otherwise it's really confusing

please update line 122 IsAssignmentToTuple_IsFalse as well